lib/active_merchant/billing/gateways/elavon.rb

```diff
@@ -1,5 +1,7 @@
 require 'active_merchant/billing/gateways/viaklix'
 require 'nokogiri'
+require 'erb'
```

Comment:
This would be the first time this is required in AM. I'm wondering if it's necessary or if there is another way to perform the url encoding...